src/components/ManaCalculator/components/RoleSection.tsx

```diff
@@ -33,12 +34,19 @@ export function RoleSection() {
         ]}
       />
       <br />
+      <label className='inlined-label'>Holded amount</label>
```

Comment:
I think it is `Held`, no?
```suggestion
      <label className='inlined-label'>Held amount</label>
```

Reply:
If I'm correct, you can probably change the var names too

Reply:
oooooops